care/middleware/time_logging_middleware.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import time
+from datetime import datetime
+
+
+class CustomFormatter(logging.Formatter):
+    def format(self, record):
+        return (
+            f"INFO {datetime.now().strftime('%Y-%m-%d %H:%M:%S')} {record.getMessage()}"
+        )
+
+
+logger = logging.getLogger("time_logging_middleware")
+logger.setLevel(logging.INFO)
+logger.propagate = False
+
+for handler in logger.handlers[:]:
+    logger.removeHandler(handler)
+
+handler = logging.StreamHandler()
```

Comment:
Al logging configurations should be handled within the django logging configuration. Remove all extra logging conf from this PR

Reply:
![image](https://github.com/user-attachments/assets/8860e453-87f8-4a6f-ae26-1e6cdbb4fc45)

Removing the config will result in logs in the above format 
is that fine ?

Reply:
You can have custom formats within django, use that to format the log statement